scripts/paths.js

```diff
@@ -1,28 +1,11 @@
-import locales from '../locales/index.js';
-import pages from '../data/pages.js';
-var localeString = ':locale';
+var locales = require('../locales/index.js');
```

Comment:
I feel like this file should live in the `data/` directory, since it's not actually an executable script